src/main/java/Duck/Storage.java

```diff
@@ -0,0 +1,77 @@
+package Duck;
+import java.io.*;
```

Comment:
Should list all imported classes explicitly

Reply:
Intellij loves to collate the imports to a *, i've changed the settings! Thanks for pointing it out!